### ISSUE TITLE
[React] [iOS] Redirect fail after login

### ISSUE THREAD

**Issue by topherauyeung** [auth0-samples/auth0-ionic-samples#538]
I've cloned this repository and it doesn't really seem to work right out of the box. I've followed the instructions (modified auth.config.ts to utilize the correct app ID, added the callback URLs and Logout URLs to my application settings, added the allowed origins, modified the custom URL scheme in the Info tab which isn't in the README) and running the app via the iOS simulator is not working. When I log in, the app does not redirect me anywhere. 

I'm seeing the following log:
```
TO JS {"url":"{redacted app ID}:\/\/{redacted domain}\/capacitor\/{redacted app ID}\/callback?
code=zAcaQU7_2FUrrjvvMzEwaa1j2tZaK4gJZHi64MCld2lnl&
state=Z2RMTXN5RXdzZEFPcW85Qnd1TkdKU0k1Vm4ydWlRM0JLZS1JOUVyUTl%2BNA%3D%3D
```
so it seems like the app is attempting to perform the redirect, but the app stays on the login page and the browser does not close. Once I manually close the browser and then click "Log in" again, it just shows a blank page in the webview.

Here is a screen recording: https://share.getcloudapp.com/ApuveA7O

**Comment by frederikprijck** [comment]
Thanks for reporting. I get a comparable, but slightly different behavior. But it doesnt seem to work neither.

Looking into it to see what can be causing it, thanks!

**Comment by thesmitchuation** [comment]
I'm experiencing the same issue exactly. Other than the info.plist is there any local configuration (i.e. xcode setup) that is missing in the docs/sample application?

**Comment by frederikprijck** [comment]
@thesmitchuation does the sample app work for you when you leave it as is and not modify the app id?

**Comment by thesmitchuation** [comment]
@frederikprijck I just got it working in my own app, and yes it did work with the sample app. I started to look at differences in the package.jsons.

Here are the things i did in my application that fixed the issue.
1. downgraded ionic cli and @ionic/angular to v6 (was on v7)
2. downgraded angular from 16 to 15
3. upgrade capacitor/core and capacitor/ios from 4.7 to 4.8
4. replaced the use of my auth service wrapping the auth0-angular service in the NgZone section in the AppComponent (where handleRedirectCallback was called).

I had tried #4 before without success. I did get success after downgrading. I didn't have success until I performed step #4 after performing the other steps.
